tests/hpc/hpc_migration.pm

```diff
@@ -133,8 +133,32 @@ sub run {
     assert_script_run("SUSEConnect --status-text");
 
     power_action('reboot', keepconsole => 1, textmode => 0);
-    #reboot and check the status again
-    assert_script_run("SUSEConnect --status-text", 180);
+
+    my $status = script_output('SUSEConnect --status-text', 180);
+    ## it is expected that all items shall be registered, since
+    ## the migration starts when all items are registered
+    if ($status =~ "Not Registered") {
+        record_info('INFO', "$status");
+        die('One of the modules or product is not registered!');
+    }
+    ## correct High Performance Computing or SUSE Linux Enterprise Server should
+    ## be there
+    if (get_var('MIGRATE_TO_HPC_PRODUCT') xor get_var('HPC_PRODUCT_MIGRATION')) {
```

Comment:
Maybe it would be a good idea to check this globally once and fail the test with some reasonable message globally in case both MIGRATE_TO_HPC_PRODUCT and HPC_PRODUCT_MIGRATION are set?